repose-aggregator/tests/performance-tests/roles/cloud_resource_create/tasks/create_network.yml

```diff
@@ -0,0 +1,8 @@
+- name: Create the Cloud Network for private communication between resources
+  local_action:
+    module: rax_network
+    credentials: ~/.raxpub
+    label: PerfNet
+    cidr: 192.168.3.0/24
+    region: IAD
+    state: present
```

Comment:
Truthfully, this network should always be present already. After all, static infrastructure like the Grafana and InfluxDB hosts already belong on it.